apps/demo-app/app/pages/charts.vue

```diff
@@ -1,7 +1,210 @@
-<script lang="ts" setup></script>
+<script lang="ts" setup>
+import { registerOnyxPlugin } from "@sit-onyx/chartjs-plugin";
+import { iconArrowSmallDown, iconArrowSmallRight, iconArrowSmallUp } from "@sit-onyx/icons";
+import type { ChartData } from "chart.js";
+import { Chart, registerables } from "chart.js";
+import { OnyxHeadline } from "sit-onyx";
+import { Bar, Line, Pie } from "vue-chartjs";
+import { useI18n } from "vue-i18n";
+
+Chart.register(...registerables);
+registerOnyxPlugin(Chart);
+
+const { t } = useI18n();
+
+/**
+ * Charts
+ */
+
+const barChartData: Ref<ChartData<"bar">> = computed(() => {
```

Comment:
```suggestion
const barChartData = computed<ChartData<"bar">>(() => {
```

We should use the generic for better typing : ) Same for the other computeds in this file